pkg/cmd/dev/generate.go

```diff
@@ -323,3 +325,41 @@ import "C"
 
 	return nil
 }
+
+func (d *dev) generateJs(cmd *cobra.Command) error {
+	ctx := cmd.Context()
+
+	args := []string{
```

Comment:
Looks like `dev generate js` doesn't call into `copyBuildOutputToWorkspace` and just does a `CopyAll`. Is that right? Is there any particular reason why it is not necessary here?

Reply:
It is right!  The original idea was to use `copyBuildOutputToWorkspace()` to copy all of the compiled typings back into the bazel workspace, but that'd require someone to rerun `./dev generate js` after every build for their IDE to see those changes.  That's a pretty tall order to ask of folks, so I made all of the TypeScript IDE tooling look in `_bazel/bin/…` for `.d.ts` files in addition to the normal `./node_modules/` location.  This allows a `./dev build` to produce new typings and for IDEs to pick them up immediately.  The only thing that did need to get copied into the workspace is a custom plugin for the JS linter: it has to be compiled before it can be used, and `eslint` doesn't seem to expose a configuration option to expand the search locations for plugins beyond the normal `node_modules` patterns.